### PR TITLE
Alters access requirements on lesser used engineering and medical supply crates.

### PR DESCRIPTION
**Pull request by TurboGay**
This PR aims to alter access requirements on the more seldom used engineering and medical supply crates while not touching the more controversial ones such as virus dishes, singularity generators and shard crates. The full list of changed crates and their respective new access requirements is as follows:

Emitter Crate - Engine Access
Field Generator Crate - Engine Access
Particle Accelerator Crate - Engine Access
Shield Generator Crate - Engine Access
Starscreen Shield Generator Crate - Engine Access
Starscreen Shield Capacitor Crate - Engine Access
Motorized Wheelchair Crate - Medical Access

Previously these crates required head access from their respective departments to open, but I personally don't see them as high risk items that would require head of staff approval to open on their own. Tested locally without issues. I would appreciate any and all feedback regarding this pr.

:cl:
 * tweak: Altered several engineering crates (emitter, field generator, particle accelerator, shield generator, starscreen shield generator and capacitor crates) access requirements from CE to engine access.
 * tweak: Altered motorized wheelchair crate access requirements from CMO to medical access.

### CODE REVIEW

**Comment by Aurx** [comment]
Which are "several engineering crate[s]"?  It's impossible to tell from the changelog.

**Comment by TurboGay** [comment]
@Aurx 

Changelog edited to better reflect that.

**Comment by TurboGay** [comment]
@MadmanMartian 

In regards to the feedback left on my last pr regarding supply crates, you mentioned that it would make heads more useless, and many seemed to agree with you. 

I don't disagree that tweaking these few crates gives them a few less things to do but the way I look at it is that heads of staff will still have a job screening the dangerous crates such as virus dishes, singularity generators and shards while allowing engineers new access to order more things (that they already typically have access to) that can be used to either fortify or fix problems on the station, in the case of the shield generators, emitters, field generators and particle accelerators which can be used to protect the station and beef up engine output respectively which falls under the job responsibilities of engineering staff and benefits practically everyone on the station.

In short, while it does take a small portion of job responsibility away from heads of staff, I think the potential benefit for lesser engineering staff will in turn improve quality of life for everyone aboard the station while not making heads of staff entirely useless like my previous pull request, by removing all crate vetting responsibilities.

I'd like to hear more feedback from you in regards to the issue to see if there's a way we might be able to meet in the middle somehow.

**Comment by Sooose** [comment]
Honestly having the engineering parts being limited to CE only is ridiculous, engineers already just open the crates up with the emitters they have. Engineers already go around their access requirements anyway, the first thing you do as an engineer is hack into the CE's office so you can unlock your hardsuit storage area. All of these packages are in secure storage anyway, where is the harm?

**Comment by chesse20** [comment]
Huge Support smacked that upvote button

**Comment by DamianX** [comment]
Yes to the engineering changes, no to the meme chair.

**Comment by FudgePucker** [comment]
Heads main role is to run their departments and macomanage their departments, not open crates.

**Comment by Kammerjunk** [comment]
Motorised wheelchair isn't the same as the tator or stackable wheelchair, mind
